scripts/sync-up-helm-chart.sh

```diff
@@ -32,7 +33,36 @@ if [ -n "${crd_diff}" ]; then
   cp -f $INSTALL_CRDS_YAML $CHART_FEDERATED_CRD_DIR/crds.yaml
 fi
 
+# Generate kubeconfig to access kube-apiserver. It is cleaned when script is done.
+cat <<EOF > ${CHART_FEDERATED_CRD_DIR}/kubeconfig
```

Comment:
Please use a temporary dir (i.e. with `mktemp -d`) to contain the kubeconfig and start etcd and the apiserver in to ensure cleanup will be automatic even if an error occurs.